src/view/shell/Drawer.tsx

```diff
@@ -569,15 +570,15 @@ function MenuItem({
   bold,
   onPress,
 }: MenuItemProps) {
-  const pal = usePalette('default')
   return (
-    <TouchableOpacity
+    <PressableScale
```

Comment:
Personal opinion: not a huge fan of this animation here. What are your thoughts on this vs a background color on press?